authors/apps/comments/serializers.py

```diff
@@ -0,0 +1,39 @@
+from rest_framework import serializers
+
+from authors.apps.articles.models import Article
+from authors.apps.authentication.serializers import UserSerializer
+from .models import Comment
+
+class RecursiveField(serializers.Serializer):
```

Comment:
@HabibSentongo, good work could you please help understand what is happening on line 7-12